# Touch detection message containing detected touch events
# Published by touch detection node

std_msgs/Header header

# Array of touch detections
orbbec_hand_detection/TouchEvent[] touches

